README.md

```
@@ -1,3 +1,5 @@
+<a href="https://flatt.tech/oss/gmo/trampoline" target="_blank"><img src="https://flatt.tech/assets/images/badges/gmo-oss.svg" height="24px"/></a>
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)